app/src/main/java/com/example/habitsmasher/listeners/SwipeListener.java

```diff
@@ -0,0 +1,46 @@
+package com.example.habitsmasher.listeners;
+
+import com.example.habitsmasher.Habit;
+import com.example.habitsmasher.HabitList;
+import com.example.habitsmasher.R;
+import com.example.habitsmasher.User;
+import com.example.habitsmasher.ui.dashboard.EditHabitFragment;
+import com.example.habitsmasher.ui.dashboard.HabitItemAdapter;
+import com.example.habitsmasher.ui.dashboard.HabitListFragment;
+import com.example.habitsmasher.ui.dashboard.RecyclerTouchListener;
+/**
+ * This is the SwipeListener class
+ * It implements the RecyclerTouchListener.OnSwipeOptionsClickListener
+ * Its purpose is to perform the onSwipeOptionClicked listener actions
+ */
```

Comment:
Also, are we going to use this class for the swipe options in HabitEventListFragment or just create a new SwipeListener class to use in there? I know that this hasn't been implemented in develop yet but I rather we have an idea of how that is going to be handled.